Changelog.md

```diff
@@ -1,6 +1,7 @@
 ### 0.4.16 (unreleased)
 
 Features:
+ * ABI: Include new field ``statemutability`` with values ``view``, ``nonpayable`` and ``payable``.
```

Comment:
More specifically: json-ABI. Also, why is `pure` not a valid option? Should we name it `stateaccess`? Or `stateaccessrestrictions`?